### DISCUSSION TITLE
Using tskit to calculate pairwise genetic similarity among samples in a VCF file

### DISCUSSION THREAD

**Discussion by loukesio** [tskit-dev/tskit#2634] [Q&A]
Dear all, 

I am new to tskit and a bit lost with the tons of information that I get. 
I would need a tiny help to kick-start my endeavour. 

I have a large vcf file which contains the variant calls after mapping. 
I want to find which samples are related and which are not and quantify this from this vcf file. 
Would it be possible to do this with tskit, and If yes, could you guide me towards it?

Sorry for my naive question. I appreciate your time

**Comment by benjeffery** [comment]
Hi @loukesio!
Thanks for your question. To get started you'll need to infer a tree sequence from your VCF. There's some detail on this at the `tsinfer` tutorial: https://tskit.dev/tsinfer/docs/stable/tutorial.html#data-example We're working on making this process easier - but that work is in progress.
Others here can pitch in with the best way to achieve your analysis, but getting to a tree sequence is the first step.

**Comment by jeromekelleher** [comment]
Hi @loukesio :wave:

It's also worth noting here that using tsinfer/tskit may not be the best approach for your data, and it's good to consider methods like plink and Hail, or even things like scikit-allel and sgkit in the Python universe. Good luck with your analysis!
